spec/models/manageiq/providers/amazon/cloud_manager_spec.rb

```diff
@@ -147,8 +149,8 @@ def create_ems_on_other_account(name)
           expect(emses.count).to eq(4)
           assert_region(emses[0], "us-east-1")
           assert_region_on_another_account(emses[1], "us-west-1")
-          assert_region_on_another_account(emses[2], "us-west-1 #{ec2_user}")
-          assert_region(emses[3], "us-west-1 1")
+          assert_region_on_another_account(emses[3], "us-west-1 #{ec2_user}")
```

Comment:
I know, those array indexes are :-1: but so is this cloudmanager spec too, with its :vhs: